examples/sagemaker-tensorflow/README.md

```diff
@@ -0,0 +1,42 @@
+# Training and Serving Merlin on AWS SageMaker
+
+With AWS Sagemaker, you can package your own models that can then be trained
+and deployed in the SageMaker environment.
+The following notebook will show you how to use NVIDIA Merlin for training and
+inference in the SageMaker environment.
+
+- [Training and Serving Merlin on AWS SageMaker]
+
+It assumes that readers are familiar wtth some basic concepts in NVIDIA Merlin,
+such as:
+
+- Using NVTabular to GPU-accelerate preprocessing and feature engineering,
+- Training a ranking model using Merlin Models, and
+- Inference with the Triton Inference Server and Merlin Models for Tensorflow.
+
+To learn more about these concepts in NVIDIA Merlin, see for example
+[Deploying a Multi-Stage Recommender System](../Building-and-deploying-multi-stage-RecSys/README.md)
+in this repository or example notebooks in
+[Merlin Models](https://github.com/NVIDIA-Merlin/models/tree/main/examples).
+
+
+## Running the Example Notebook
+
+You can run the example notebook with Docker containers.
+See [Running the Example Notebooks](../README.md#running-the-example-notebooks)
```

Comment:
can you be more specific with the container? I think your example nb can only be run with `merlin-tensorflow:22.XX` container. May be you can rephrase it as `You can run the example notebook with the latest stable merlin-tensorflow container` and link the container tag on NGC?

Reply:
I find the container tags tiresome.  My sugg is to name the `nvcr.io/nvidia/merlin/merlin-tensorflow:nightly` container and indicate that for production workloads, NVIDIA offers containers that are tagged for each release.

I gave it a whirl at https://github.com/NVIDIA-Merlin/HugeCTR/tree/main/sparse_operation_kit/notebooks#prerequisites.